src/main/kotlin/org/rust/lang/core/resolve/ImplLookup.kt

```diff
@@ -55,6 +56,9 @@ class ImplLookup(private val project: Project, private val items: StdKnownItems)
     val fnOutputParam by lazy(NONE) {
         RsLangItemIndex.findLangItem(project, "fn_once")?.let { findFreshAssociatedType(it, "Output") }
     }
+    private val copyTrait: RsTraitItem? by lazy(NONE) {
```

Comment:
Is it possbile to make this more generic?

Reply:
Hm, could you elaborate a bit?

Reply:
I would change it to a function, than you say copy or debug, for example: driveTrait("Debug")...

Reply:
Yeah, that makes sense, and we should definitely do it once we handle more then one derived trait specially. But for a single use case, I usually prefer not to create an abstraction upfront.

Reply:
Okay